algorithm/snow/fv3jedi_land_ensrecenter.yaml.j2

```diff
@@ -0,0 +1,63 @@
+geometry:
+  fms initialization:
+    namelist filename: ./fv3jedi/fmsmpp.nml
+    field table filename: ./fv3jedi/field_table
+  akbk: ./fv3jedi/akbk.nc4
+  layout:
+  - {{ layout_x }}
+  - {{ layout_y }}
+  npx: {{ npx_ges }}
+  npy: {{ npy_ges }}
+  npz: {{ npz_ges }}
+  field metadata override: ./fv3jedi/fv3jedi_fieldmetadata_restart.yaml
+date: "{{ atm_background_time_iso }}"
+variables: [totalSnowDepth]
+increment mask:
+  variable: [slmsk]
+  maxvalue: 0.99 
+deterministic increment:
+  datetime: "{{ atm_background_time_iso }}"
+  filetype: fms restart
+  state variables: [snodl,vtype,slmsk]
+  datapath: ./anl
```

Comment:
If this is a read it might also need a `filename is datetime templated: true` flag